### PR TITLE
Feat/irv 10 fix scroll bars

### PR DESCRIPTION
**Pull request by tnoelcke**
### Updates Made
This pr removes extra scroll bars form the item bank viewer as well as fixing some style issues.

### Contributing developer checklist
- [x] I've updated source branch with the latest changes from dev.
- [x] I've added/changed unit tests for components with functionality.
- [x] I've added/updated storybook for any component that I've changed.
- [ ] I've reviewed each jsdoc header in regards to the changes that I've made and updated them.

### Reviewing developer checklist
- [x] I've pulled this branch to my local machine.
- [x] I've inspected the changes on storybook.
- [x] I've run the test suite and all tests have passed

### CODE REVIEW

File: src/ItemBank/ItemBankContainer.tsx [hunk 1]
```diff
@@ -213,6 +215,57 @@ export class ItemBankContainer extends React.Component<
...
+    if (
+      !state.currentItem &&
+      !state.nextItem &&
+      !state.previousItem &&
+      this.state.currentItem
+    ) {
+      this.props.resetUrl();
+    } else if (state.items !== this.state.items && state.items.length > 1) {
+      this.handleChangeViewItem();
+    }
```
Comment (iglenwoo) [inline src/ItemBank/ItemBankContainer.tsx]: What will happen if else?

File: src/ItemBank/ItemBankContainer.tsx [hunk 2]
```diff
@@ -213,6 +215,57 @@ export class ItemBankContainer extends React.Component<
...
+      if (itemsAreEqual(state.items[key], currentItem)) {
+        if (state.previousItem) {
+          currentItem = state.previousItem;
+        } else if (state.nextItem) {
+          currentItem = state.nextItem;
+        } else if (state.items.length === 2) {
+          currentItem = undefined;
+          nextItem = undefined;
+          previousItem = undefined;
+        }
```
Comment (iglenwoo) [inline src/ItemBank/ItemBankContainer.tsx]: I think without `else` here, this logic can cause a problem.

Fellow this procedure.
1. Put below items in CSV Entry and Apply them
itemreviewviewer,187,3001,SIW
iat-uat,47320,SIW
2. Remove the lowest item iat-uat,47320,SIW.
3. You can see some weird thing happens.

Comment (tnoelcke) [reply src/ItemBank/ItemBankContainer.tsx]: I've fixed this problem. It has more to do with there being an invalid item as the current item.

File: src/ItemBank/ItemBankContainer.tsx (hunk 1 above)
Comment (tnoelcke) [reply src/ItemBank/ItemBankContainer.tsx]: in the else branch we don't want to do any thing. This is a life cycle method that is called when the state is updated. We only want to do something when the stat is actually changing and this ensures that this is true for the case where we add an item and for the case where we are deleting the last item.

File: src/ItemBank/ItemBankContainer.tsx [hunk 3]
```diff
@@ -231,9 +284,11 @@ export class ItemBankContainer extends React.Component<
         })
         .catch(e => {
           this.onError(e, () => {
-            index = items.findIndex(i => i === currentItem);
-            items[index].valid = false;
-            this.setState({ items }, this.updateNavigationItems);
+            if (items.length > 1) {
```
[the remark this answers is not in the document]
Comment (tnoelcke) [reply src/ItemBank/ItemBankContainer.tsx]: I try not to return from an if statement. I prefer to have return statements be the last line in a function.

**Pull request merged**